erpnext/erpnext_integrations/doctype/leaflink_settings/leaflink_settings.py

```diff
@@ -175,7 +175,7 @@ def sales_order_from_leaflink(**args):
 		except Exception as e:
 			integration_request.update({
 				"status": "Failed",
-				"error": e
+				"error": str(e)
```

Comment:
```suggestion
				"error": str(e)
```
use frappe.utils.cstr